proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -415,3 +419,12 @@ message PayloadRepositoryLabelMoved {
   // from_commit_id is the id of the commit on which the label was moved from.
   string from_commit_id = 7;
 }
+
+message PayloadServerBreakingChangesChanged {
+  // enabled tracks whether breaking changes were enabled or disabled.
```

Comment:
@unmultimedio, is the audit event meant to capture the old state, or is it sufficient to record the new changes?

Reply:
We've been tracking both old and new properties, e.g.: https://github.com/bufbuild/buf/blob/8df996524679da8fca7c006c957fb4ce7cf0f3b2/proto/buf/alpha/audit/v1alpha1/event.proto#L182-L195

Although this being a boolean, I don't know, feels weird.